test/e2e/apiserver.go

```diff
@@ -131,3 +131,87 @@ var _ = framework.KubeDescribe("API Server webhook tests (ignored namespace)", f
 		Expect(err).NotTo(HaveOccurred())
 	})
 })
+
+var _ = framework.KubeDescribe("API Server webhook tests for pods (enabled namespace)", func() {
```

Comment:
This should have a test for changing the image later (which we've missed).

Reply:
This was added: https://github.com/zalando-incubator/kubernetes-on-aws/blob/bbeda1afea5de87d117ecaa9bfe09082c028c53d/test/e2e/apiserver.go#L270-L281